### ISSUE TITLE
firebase-bigquery: "deployment location" is misleading since BigQuery table was in different location.

### ISSUE THREAD

**Issue by rphlmr** [firebase/extensions#96]
### [READ] Step 1: Are you in the right place?

Issues filed here should be about bugs for a **specific extension in this repository**.
If you have a general question, need help debugging, or fall into some
other category use one of these other channels:

- For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
  with the firebase tag.
- For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
  google group.
- To file a bug against the Firebase Extensions platform, or for an issue affecting multiple extensions, please reach out to
  [Firebase support](https://firebase.google.com/support/troubleshooter/contact/) directly.

### [REQUIRED] Step 2: Describe your configuration

- Extension name: **\_** `firebase-bigquery`
- Configuration values (redact info where appropriate):
  - location set to `europe-west1`

#### Steps to reproduce:

I deployed firebase-bigquery extension, with location set to `europe-west1`. But in bigquery my data is located in US.

##### Expected result

Deployed in the location set during initialization

##### Actual result

Wrong location

**Comment by rphlmr** [comment]
Checking in bigquery directly, `europe-west1` is not available. But it is proposed in the extension configuration.

**Comment by Salakar** [comment]
Hey @rphlmr - thanks for the report, have checked the code and it looks like `initializeDataset` is not setting the dataset location over at `this.bq.database(*)`:

https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts#L103


```ts
dataset(id: string, options?: DataSetOptions): Dataset;
```

```ts
export interface DataSetOptions {
    location?: string; // probably want it to set this
}
```

Not sure the best approach on fixing this, probably adding an additional param to specify the BQ location but constrain it to the [available regional & multi-regional locations](https://cloud.google.com/bigquery/docs/locations) - could have it default to `US` (multi-regional) so it's a non breaking change for existing deploys.

cc @laurenzlong - do you have any thoughts on this?

**Comment by laurenzlong** [comment]
Yes I do think the best approach is the revise the current LOCATION param to indicate it only affects where the Cloud Functions are deployed to, and add a new param for specifying where BigQuery data lives. But I will review the options with the team first. @rphlmr would that solve your use case?

**Comment by rphlmr** [comment]
Hi @laurenzlong,

At first, I think it's not clear that Location is for the cloud function location.
I really thought it was for both :D 
For people dealing with "GDPR" it's important to be able to choose a location for our dataset.
What you suggest seems perfect.

What I did to solve this was to create by hand my dataset. (and for the data already in production, I moved my data and reconfigure my functions)
